### DISCUSSION TITLE
Welcome to companion Discussions!

### DISCUSSION THREAD

**Discussion by dnmeid** [bitfocus/companion#2386] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by T-BoneFirefly** [comment]
Hey all,

My name is Hunter and I have been using Companion to automate lots of operations for an arena in the US. We use it to control all kinds of hardware from PTZ cameras to IPTV systems to... I mean almost everything. We have automated tasks to be much less difficult and time-consuming.

Maybe the best thing about Companion is that it has allowed me to rip Crestron systems out of multiple rooms- for that alone I believe all of you deserve the Nobel Peace Prize.

Anyway, happy to see this discussion thread open up. I'm not on Facebook, so this is a welcome opportunity to engage with the community! Looking forward to chatting with you all.

**Comment by Prathamesh-Shigwan** [comment]
i want to run and modify companion project on my local machine but i getting errors

PS C:\Users\Administrator\Desktop\test2\companion-master> npm install
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: companion@3.0.1
npm ERR! notsup Not compatible with your version of node/npm: companion@3.0.1
npm ERR! notsup Required: {"npm":"please-use-yarn","yarn":"^1.22","node":">=18.12 <19"}
npm ERR! notsup Actual:   {"npm":"10.1.0","node":"v18.17.1"}

what should i do is there any manual that help to run companion-master project on local machines

**Reply by Julusian** [reply]
https://github.com/bitfocus/companion/discussions/2576

**Comment by Prathamesh-Shigwan** [comment]
where can i find companion@3.0.0 package there is only one package 
PS C:\Users\Administrator\Desktop\test2\companion-master> npm show companion

companion@0.0.2 | MIT | deps: none | versions: 1
Load vanilla javascript in node.

dist
.tarball: https://registry.npmjs.org/companion/-/companion-0.0.2.tgz
.shasum: 8a618cbbd501ee43cedf871219bb4e6539a21235
.integrity: sha512-tV6MQ19VM4s5sSiZf0eNsnLT+qZkOeWltzD0RpNMoNoKaRj9gDlydc5OQMyKs461BqFLhm2eJncAtoJwIpNFIw==

maintainers:
- young-steveo <young.steveo@gmail.com>

dist-tags:
latest: 0.0.2

published over a year ago by young-steveo <young.steveo@gmail.com>

**Reply by Julusian** [reply]
We don't publish the project to npm, that is showing you something else

**Comment by Southcrest** [comment]
hello i need help getting companion to work with spotify

**Reply by Southcrest** [reply]
i can't seem to get the approval code to show up in the blue box, please advise

**Reply by thedist** [reply]
Then raise an issue on the Spotify modules repo https://github.com/bitfocus/companion-module-spotify-remote

**Reply by Southcrest** [reply]
awesome thanks
